applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -110,7 +110,10 @@ def GetDefaultSettings(cls):
             "reform_dofs_at_each_step": false,
             "line_search": false,
             "compute_reactions": true,
-            "block_builder": true,
+            "block_builder" : true,
```

Comment:
same comment as in the other PR
It would be easier to discuss the changes in StructuralMechanics in a separate PR after the ones in the Core are merged

Reply:
+1

Reply:
to be coherent with the rest of the code, this block should be

nevertheless i understand that maybe it is not a good idea to do this change as part of the current PR

~~~json
"builder_and_solver_settings" : {
                "builder_and_solver_type" : "block",
                "diagonal_values_for_dirichlet_dofs" : "use_max_diagonal",
                "silent_warnings"                    : false
            },
~~~

Reply:
I thought that, but it will break many parts, we should do that in a independent PR

Reply:
Guys we can vote but I will vote no for changing this also in this PR, since this can be done with little effort in a future PR 
;)

Reply:
I agree about NOT changing it in this PR, nevertheless i think it is a todo for the future

Reply:
I understand wrongly, do I revert?, the modification is retrocompatible

Reply:
well, the second line of my sentence was "i understand that maybe it is not a good idea to do this change as part of the current PR" ... and both you, Philipp and me agree on the essence that this would be good, but that it should not go in here as it is API breaker.

I would not put this change as part of this PR, but rather in the PR with the factory for strategies...

Reply:
Well, this is easy to solve, i just need to revert a commit